### PR TITLE
feat: reactive title bar based on user's theme

### LINKED ISSUE #2139: Safari PWA title bar should be black
### Describe the feature

Change the title bar to black instead of the current red color that is very distracting

1E99E588-62D1-4D6F-9D3C-CE70294B225B

### Why would this be useful to add?

Using the PWA on iOS diverts my eyes to the top bright red color and it’s very distracting

### Concept(s)

_No response_

### Additional context

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by rramiachraf**
Fixes #2139

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -34,6 +34,12 @@ export default {
             if (themePref == "auto") this.theme = darkModePreference.matches ? "dark" : "light";
             else this.theme = themePref;
 
+            // Change title bar color to white
+            if (this.theme === "light") {
+                const themeColor = document.querySelector("meta[name='theme-color']");
+                themeColor.setAttribute("content", "#FFF");
+            }
```
Comment (FireMasterK) [inline src/App.vue:41]: Don't we need to set it back to dark if the theme changes back to dark? (eg: when using auto mode)

Comment (rramiachraf) [reply src/App.vue:41]: The `theme-color` meta tag by default uses the dark theme, therefore if dark theme is set the condition would be false and the theme color won't be set which means a fallback to the default value.

Comment (rramiachraf) [reply src/App.vue:41]: Another approach would be injecting the meta tag conditionally based on user's theme.

Comment (FireMasterK) [reply src/App.vue:41]: What if this scenario:

- The user has auto theme mode and is currently using dark mode
- The user changes their theme from their system preferences to light, and then to dark (without reloading the page)

Won't it remain light in such a scenario?

Comment (rramiachraf) [reply src/App.vue:41]: You're absolutely right, forgot about this scenario.
Next commit will fix this.

**Review by FireMasterK — APPROVED** [review]
Thank you!

**Pull request merged**